CareKit.xcodeproj/project.pbxproj

```diff
@@ -1482,10 +1483,11 @@
 		8677EE041C9775EB00588CD6 /* OCKCarePlanStore.xcdatamodeld */ = {
 			isa = XCVersionGroup;
 			children = (
+				96AED2731EEC933400C98B5B /* OCKCarePlanStore v 3.xcdatamodel */,
 				185BC3A31D81DF0F001AB0A9 /* OCKCarePlanStore v2.xcdatamodel */,
 				8677EE051C9775EB00588CD6 /* OCKCarePlanStore.xcdatamodel */,
 			);
-			currentVersion = 185BC3A31D81DF0F001AB0A9 /* OCKCarePlanStore v2.xcdatamodel */;
+			currentVersion = 96AED2731EEC933400C98B5B /* OCKCarePlanStore v 3.xcdatamodel */;
```

Comment:
Would you mind keeping the same format here on the data models? So OCKCarePlanStore v3 without the space between v and 3?

Reply:
@kelseydedoshka-apple Of course! I've pushed that update. (Not sure how that space got in there, I was pretty sure I just used Xcode to add a new model version, but oh well!)